src/Engine/Graphics/Renderer/OpenGLRenderer.cpp

```diff
@@ -4135,10 +4136,7 @@ void OpenGLRenderer::DrawIndoorFaces() {
 
                     // check if this face is visible through current portal node
                     if (pCamera3D->CullFaceToFrustum(static_vertices_buff_in, &uNumVerticesa, static_vertices_calc_out, portalfrustumnorm, 4)) {
-                        face->uAttributes |= FACE_SeenByParty;
-
-                        // check face is towards camera
-                        if (pCamera3D->is_face_faced_to_cameraBLV(face)) {
+                        if (true) {
```

Comment:
Might as well tidy this up now

Reply:
I plan to do a bigger cleanup so that would be just noise.